src/test.zig

```diff
@@ -3,4 +3,5 @@ const testing = std.testing;
 
 test {
     testing.refAllDeclsRecursive(@import("main.zig"));
+    testing.refAllDeclsRecursive(@import("build/solve_functions.zig"));
```

Comment:
This theoretically shouldn't be needed once everything is imported even indirectly by main. If we want to help avoid things breaking, it would probably be good to add this for all passes and for base.zig, collections.zig, and coordinate.zig

Reply:
Ya, I can add those in.

The weird thing is that `main` _does_ import `coordinate`, which itself imports `solve_functions`. Maybe every module down the chain needs:

```zig
test {
    testing.refAllDeclsRecursive(@This));
}
```